src/components/menu/Menu.tsx

```diff
@@ -0,0 +1,16 @@
+// (C) 2007-2018 GoodData Corporation
+import MenuTopLevel, { IMenuTopLevelProps } from './MenuTopLevel';
+import Menu, { IMenuProps } from './SubMenu';
+
+/*
+NOTE (2018-11-20):
+    Not tested on mobile (out of scope of original ticket). Events might need to
+    be changed/added, eg.: start using onTouchStart and others instead of just onClick.
+*/
+
+export {
+    IMenuTopLevelProps,
+    MenuTopLevel,
```

Comment:
Why this file exists?
 
I would say: replace this file content with MenuTopLevel component. And I guess `Menu` as name is enough instead of `MenuTopLevel`.

Reply:
It is here so you have one file where you can import all relevant components.
So you can do 

```js
import { Menu, SubMenu } from '../Menu';
```

Instead of

```js
import { Menu } from '../Menu';
import { SubMenu } from '../SubMenu';
```

Also in the following ticket there will be several presentational components like menu item or separator or checkbox, so my idea for API, at the moment, is that all these components will be available in one place to import from.

Reply:
Updated, index file removed.